src/components/layout/PageTitle.tsx

```diff
@@ -8,17 +9,19 @@ const PageTitle = ({
   title: ReactNode;
   actions?: ReactNode;
 }) => {
+  const isTiny = useIsMobile('sm');
+
   return (
     <Stack
-      gap={3}
-      direction='row'
+      gap={isTiny ? 1 : 3}
+      direction={isTiny ? 'column' : 'row'}
       justifyContent={'space-between'}
+      width={isTiny ? 'fit-content' : '100%'}
       sx={{
-        mb: 3,
-        pr: 1,
```

Comment:
I removed padding-right here, since it seemed to add unnecessary spacing, but not sure if that was the right move?